### DISCUSSION TITLE
Roadmap Feedback - How are we doing?

### DISCUSSION THREAD

**Discussion by jfuss** [aws/aws-sam-cli#4689] [Feedback]
Leave a reaction or a comment to express feedback on our community engagement. You can find our [public roadmap here](https://github.com/aws/aws-sam-cli/wiki/AWS-SAM-CLI-Roadmap).

This was migrated from Issues. Original Issue can be found [here](https://github.com/aws/aws-sam-cli/issues/3632)

**Comment by kaihendry** [comment]
Made a list of things I'd hope you would prioritise https://dabase.com/blog/2023/sam-developer-experience/
